rhoas-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+event-bridge.rhoas.mgmt-api.host=https://api.openshift.com
+event-bridge.rhoas.instance-api.host=http://localhost
+event-bridge.rhoas.sso.red-hat.auth-server-url=https://sso.redhat.com/auth/realms/redhat-external
+event-bridge.rhoas.sso.red-hat.refresh-token=dummy-refresh-token
+event-bridge.rhoas.sso.mas.auth-server-url=https://identity.api.openshift.com/auth/realms/rhoas
```

Comment:
Please use variables (with default value) for each of them

Reply:
I removed the default values and listed all the configuration variables in the `README` file :+1: